src/hostapi/wdmks/pa_win_wdmks.c

```diff
@@ -163,15 +163,9 @@ Default is to use the pin category.
 #define DYNAMIC_GUID(data) {data}
 #define _NTRTL_ /* Turn off default definition of DEFINE_GUIDEX */
 #undef DEFINE_GUID
-#ifdef __clang__ /* clang-cl: avoid too many arguments error */
-  #define DEFINE_GUID(n, ...) EXTERN_C const GUID n = {__VA_ARGS__}
-  #define DEFINE_GUID_THUNK(n, ...) DEFINE_GUID(n, __VA_ARGS__)
-  #define DEFINE_GUIDEX(n) DEFINE_GUID_THUNK(n, STATIC_##n)
-#else
-  #define DEFINE_GUID(n, data) EXTERN_C const GUID n = {data}
-  #define DEFINE_GUID_THUNK(n, data) DEFINE_GUID(n, data)
-  #define DEFINE_GUIDEX(n) DEFINE_GUID_THUNK(n, STATIC_##n)
-#endif /* __clang__ */
+#define DEFINE_GUID(n, ...) EXTERN_C const GUID n = {__VA_ARGS__}
```

Comment:
Are there any compilers where this variadic code will fail?

Maybe the original #ifdef on line 166 should be extended with defined(__MSVC__) or something like that.

Reply:
Yes, we are already inside a `#ifdef _MSC_VER` block!

Reply:
The variadic code should be avoided with MSVC (standard non-clang front end). That's the whole point of that code